### PR TITLE
Add missing delta-debug documentation

### LINKED ISSUE #151: Delta-debug section is missing in the README

### PR DESCRIPTION
**Pull request by pablooliveira**
Fix #151

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -507,6 +514,75 @@ For more information about `vfc-vtk.py`, please use the online help:
 $ postprocess/vfc-vtk.py --help
 ```
 
+## Pinpointing errors with Delta-Debug
+
+[Delta-Debug (DD)](https://www.st.cs.uni-saarland.de/dd/) is a general bug
```
Comment (mistoan) [inline README.md:519]: a generic bug-reduction?

File: README.md [hunk 2]
```diff
@@ -507,6 +514,75 @@ For more information about `vfc-vtk.py`, please use the online help:
...
+reduction method that allows to efficiently find a minimal set of conditions
+that trigger a bug. In this case, we are going to consider the set of
+floating-point instructions in the program. We are using DD to find a minimal
+set of instructions responsible for the instability in the output.
```
Comment (mistoan) [inline README.md:523]: the possible instability?

File: README.md [hunk 3]
```diff
@@ -507,6 +514,75 @@ For more information about `vfc-vtk.py`, please use the online help:
...
+
+By testing instructions sub-sets and their complement, DD is able to find
```
Comment (mistoan) [inline README.md:525]: instruction sub-sets

File: README.md [hunk 4]
```diff
@@ -507,6 +514,75 @@ For more information about `vfc-vtk.py`, please use the online help:
...
+developed in the [verrou](https://github.com/edf-hpc/verrou) project.
+
+To use delta-debug, we need to write two scripts:
+
+   - A first script ``ddRun <output_dir>``, is responsible for running the
+     program and writing its output inside the ``<output_dir>`` folder.
+
+   - A second script ``ddCmp <reference_dir> <current_dir>``, takes as
+     parameter two folders including respectively the outputs from a reference
+     run and from the current run. The `ddCmp` script must return with a
```
Comment (mistoan) [inline README.md:538]: must return success

File: README.md [hunk 5]
```diff
@@ -507,6 +514,75 @@ For more information about `vfc-vtk.py`, please use the online help:
...
+tolerance of the application domain; therefore it is hard to provide a generic
+script that fits all cases. That is why we require the user to manually write
+these scripts.  Once the scripts are written, the Delta-Debug session is
+launched using the following command:
+
+```bash
+$ VFC_BACKENDS="libinterflop_mca.so -p 53 -m mca" vfc_ddebug ddRun ddCmp
+```
+
+The `VFC_BACKENDS` variable selects the noise model among the available
```
Comment (mistoan) [inline README.md:556]: This is a bit ambiguous on whether the rest of the back-ends can be used as well.
Maybe specify it explicitly?

File: README.md [hunk 6]
```diff
@@ -507,6 +514,75 @@ For more information about `vfc-vtk.py`, please use the online help:
...
+instrumented with the noise backend during the run).
+
+The union of the _"culprit"_ instructions can also be found in
+`dd.line/rddmin-cmp/dd.line.exclude`.
+
+A full example demonstrating delta-debug usage can be found in the
+[tutorial](https://github.com/verificarlo/verificarlo/wiki/Tutorials) and in
+the `tests/test_ddebug_archimedes`.
+
+As an example, in the `test_ddebug_archimedes`, two ddmin sets are found:
```
Comment (mistoan) [inline README.md:573]: No mention of the integration with the text editor.
Should point it out.

**Comment by pablooliveira** [comment]
@mistoan I made the suggested changes, thanks.

**Pull request merged**